app/src/main/java/ch/epfl/sdp/blindly/main_screen/match/cards/CardStackAdapter.kt

```diff
@@ -36,6 +45,11 @@ class CardStackAdapter(
     override fun onCreateViewHolder(parent: ViewGroup, viewType: Int): ViewHolder {
         context = parent.context
         val inflater = LayoutInflater.from(context)
+        //Initialize the mediaPlayers' list
+        for (i in 0 until itemCount) {
```

Comment:
Would this be a problem in terms of scaling?  Just asking to get more knowledge.

Reply:
Yes, it would probably become a problem... But I couldn't find a better way since the recordings don't come in order, I have to store them in the correct position so the array has to be pre-initialized. To fix this I think we should download matches partially every N swipes, so that the loading doesn't take too much time and the mediaPlayer's initialization isn't problematic either... But as it is right now I can't of a better way

Reply:
I see, thanks for the explanation!